### PR TITLE
Ports pineapple juice, creme de coconut, pina colada and painkiller.

### PR DESCRIPTION
**Pull request by Dingo-Dongler**
## About The Pull Request

Ports https://github.com/tgstation/tgstation/pull/48783 from tgstation. 

## Why It's Good For The Game

I like the tropical theming for the bar, and I want the drinks to match.

## Changelog
:cl:
add: Added pineapple juice, creme de coconut, pina coladas and painkillers.
/:cl:

### CODE REVIEW

**Review by Archanial — APPROVED** [review]

**Comment by Archanial** [comment]
Make wiki changes and ping me

**Comment by ike709** [comment]
Did you ever add these to the wiki?

**Comment by Dingo-Dongler** [comment]
I tried, but every time i upload the pina colada image it says im uploading the painkiller image again. shit sucks

File: code/modules/hydroponics/grown/pineapple.dm [hunk 1]
```diff
@@ -30,5 +30,6 @@
 	filling_color = "#F6CB0B"
 	w_class = WEIGHT_CLASS_NORMAL
 	foodtype = FRUIT | PINEAPPLE
+	juice_results = list(/datum/reagent/consumable/pineapplejuice = 0)
```
Comment (PowerfulBacon) [inline code/modules/hydroponics/grown/pineapple.dm:33]: Is this meant to be 0?

**Review by PowerfulBacon — APPROVED** [review]

**Comment by DatBoiTim** [comment]
Wiki changes for bartending recipes are needed.

#6185

**Pull request merged**